tests/datatypes/test_bytearray.py

```diff
@@ -40,6 +40,15 @@ def test_islower(self):
         """)
         # self.assertCodeExecution("""""")
 
+    def test_isspace(self):
+        self.assertCodeExecution("""
+            print(bytearray(b'testupper').isspace())
+            print(bytearray(b'test isspace').isspace())
+            print(bytearray(b' ').isspace())
+            print(bytearray(b'').isspace())
+            print(bytearray(b'\x46').isspace())
```

Comment:
Can you please add these cases?

```
b'   \t\t' # several spaces mixed with tabs
b' \t\n\r\x0b\f' # others kinds of space characters
```

See: https://docs.python.org/3/library/stdtypes.html#bytearray.isspace